tests/test__mongomock.py

```diff
@@ -1188,6 +1188,9 @@ def test__update_one(self):
                                upsert=True)
         self.cmp.compare.find()
 
+        self.cmp.compare_exceptions.update_one({}, {'$set': {}})
+        self.cmp.compare_exceptions.update_one({'a': 'does-not-exist'}, {'$set': {}})
```

Comment:
please, also test on upsert (although upsert shouldn't be used with `$set`, maybe the output error is not the right one)

Reply:
Done.